pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -165,13 +156,17 @@ func toAntreaIPBlockForCRD(ipBlock *secv1alpha1.IPBlock) (*networking.IPBlock, e
 }
 
 // getTierPriority retrieves the priority associated with the input Tier name.
-// If the Tier name is empty, by default, the lowest priority Application Tier
-// is returned.
-func getTierPriority(tier string) networking.TierPriority {
+func (n *NetworkPolicyController) getTierPriority(tier string) *uint32 {
 	if tier == "" {
-		return antreatypes.TierApplication
+		return nil
```

Comment:
So the agent will be responsible for translating `tierPriority == nil` to map to default tier?